wfe2/verify.go

```diff
@@ -315,6 +351,15 @@ func (wfe *WebFrontEndImpl) extractJWK(jws *jose.JSONWebSignature) (*jose.JSONWe
 	return key, nil
 }
 
+// jwsKeyIDToAccountID returns the KeyID string from a JWS. It should be called
+// only using JWS from `parseJWS` to ensure the correct number of Signatures are
+// present.
+func (wfe *WebFrontEndImpl) jwsKeyIDToAccountID(jws *jose.JSONWebSignature) string {
+	header := jws.Signatures[0].Header
```

Comment:
These assignments seem unnecessary, am I missing something or could this just be `return jws.Signatures[0].Header.KeyID`?